packages/nx/src/plugins/js/index.ts

```diff
@@ -51,7 +52,10 @@ export const createNodes: CreateNodes = [
     }
 
     const lockFilePath = join(workspaceRoot, lockFile);
-    const lockFileContents = readFileSync(lockFilePath).toString();
+    const lockFileContents =
+      packageManager !== 'bun'
+        ? readFileSync(lockFilePath).toString()
+        : execSync(lockFilePath).toString();
```

Comment:
Shouldn't it be `bun bun.lockb` as the command?

Reply:
Addressed in the other comment